### PR TITLE
Feature/sku selector

### PR DESCRIPTION
**Pull request by brenoc**

### CODE REVIEW

File: src/components/product/SkuSelector.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  changeAvailability = function (){
+    if(this.mock.skus[0].availability > 0)
+      return v-dream__size-selector;
+    else
+    return v-dream__size-selector--unavailable;
+  },
+
+  checkVariations = function (){
+    createWishlistComponent() {
+   if (this.skus.length()<=1) {
```
Comment (brenoc) [inline src/components/product/SkuSelector.jsx]: Didn't the linter throw an error here?

It should be:
```js
if (this.skus.length() <= 1) {
```

File: src/components/product/SkuSelector.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        id: 444
+        size: 'G'
+        availability: 2
+      }
+    ]
+};
+
+class SkuSelector extends React.Component {
+  changeAvailability = function (){
+    if(this.mock.skus[0].availability > 0)
```
Comment (brenoc) [inline src/components/product/SkuSelector.jsx]: The same as the other comment:

```js
if (this.mock.skus[0].availability > 0)
```
The thing here is the space between `if` and the `(`.

File: src/components/product/SkuSelector.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      {
+        id: 444
+        size: 'G'
+        availability: 2
+      }
+    ]
+};
+
+class SkuSelector extends React.Component {
+  changeAvailability = function (){
```
Comment (brenoc) [inline src/components/product/SkuSelector.jsx]: You can use a syntax sugar here:
```js
changeAvailability() {
```

**Comment by brenoc** [comment]
Good job! That's the way!

File: src/components/product/SkuSelector.jsx (hunk 3 above)
Comment (tamorim) [inline src/components/product/SkuSelector.jsx]: Also, when defining functions, usually it goes like this:

```javascript
checkAvailability = function() {
  // ...
}
```
The parenthesis stays right after the `function` keyword and there's a space before the opening bracket :)

Only conventions.

File: src/components/product/SkuSelector.jsx (hunk 1 above)
Comment (tamorim) [inline src/components/product/SkuSelector.jsx]: The `return` on `else` is also not correctly indented.
Besides, the `else` line can be deleted, if it goes inside de `if`, it doesn't continue anyway.

**Pull request merged**